packages/app/src/systems/Bridge/hooks/useWithdrawDelay.tsx

```diff
@@ -0,0 +1,41 @@
+import { useQuery } from '@tanstack/react-query';
```

Comment:
I think we don't need to over load the machines with this kind of query, as they are not really crucial for the flow. I think this is an approach we should start doing, to avoid unnecessary complexity.